news/_posts/performance/2020-12-07-performance-and-prestashop.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  "Performance and PrestaShop"
+subtitle: "Sorry, Goose, but it's time to buzz the tower."
+date:   2020-12-04 9:00:00
+authors: [ Claude-ArnaudPerrot ]
+icon: icon-airplane
+tags:
+ - performance
+ - whitepaper
+---
+
+Let's start discussing performance and tuning with PrestaShop! 
+
+# Performance (& PrestaShop)
+
+You may have heard of it, but we'll soon release a White Paper, which is all about PrestaShop's performance, how to benchmark it and how it behaves under loads. And what to do to keep it responding under 300ms.
+
+Still, there are a number of matters that won't be covered in this White Paper - hopefully the first in a series, fingers crossed - that we'd love to shine a light on.
+
+Because, funny thing is, we often talk about performance as something we all understand and it is mostly true: we share the same expectations. 
+
+Though, we're not always on line with its implications.
+
+So, here we go!
+
+## Performance is subjective
+
+It may seem counter-intuitive coming from a technical environment, either from web development or any other domain, but still performance is all about capabilities.
+
+Just to illustrate, let's take this well-known number:
+
+![NotANumber](https://upload.wikimedia.org/wikipedia/commons/5/56/Answer_to_Life.png)
+
+Is it high? Or is it low? But compared to what? In which context?
+
+Same thing, if I'm told that in a [GCP](https://cloud.google.com/gcp/) environment, the maximum write sustained IOPS per GB for a regional SSD drive is 30: does it mean it performs? Is it good or...? 
+
+But let's be even more precise: in our White Paper scope, we assumed a web page should load *under* 300ms to be deemed acceptable for our users. Hence, we based our benchmarks on this value - and results above this response time where discarded as unacceptable.
```

Comment:
300ms is however a number often mentioned as best practice ? like "after 300ms user leaves the site" ?

Reply:
Well, that's the point : 300ms does not mean anything without context, and the context will evolve with time.
Hence, it's subjective ;)